src/services/config/ApplicationInsights.js

```diff
@@ -5,15 +5,20 @@ import { DistributedTracingModes } from '@microsoft/applicationinsights-web';
 import { APPLICATION_INSIGHTS_INSTRUMENTATION_KEY } from '../../config/AppInstance.js';
 
 // Application Insight configuration
-export const APPLICATION_INSIGHTS_CONFIG = {
-  name: 'Web Application Sample',
-  config: {
-    instrumentationKey: APPLICATION_INSIGHTS_INSTRUMENTATION_KEY,
-    disableFetchTracking: false,
-    enableCorsCorrelation: true,
-    enableRequestHeaderTracking: true,
-    enableResponseHeaderTracking: true,
-    enableAutoRouteTracking: true,
-    distributedTracingMode: DistributedTracingModes.AI_AND_W3C
+export const getApplicationInsightsConfig = () => {
+  if (APPLICATION_INSIGHTS_INSTRUMENTATION_KEY) {
+    return {
+      name: 'Web Application Sample',
+      config: {
+        instrumentationKey: APPLICATION_INSIGHTS_INSTRUMENTATION_KEY,
+        disableFetchTracking: false,
+        enableCorsCorrelation: true,
+        enableRequestHeaderTracking: true,
+        enableResponseHeaderTracking: true,
+        enableAutoRouteTracking: true,
+        distributedTracingMode: DistributedTracingModes.AI_AND_W3C
+      }
+    };
   }
+  return undefined;
```

Comment:
This file is not supposed to have any logic in it, the check of the `APPLICATION_INSIGHTS_INSTRUMENTATION_KEY` value should be done in the service, to keep the configuration file as simple as possible.